File: no-sdk/dotnet/AwsCredentials.cs
public static class AwsCredentials
{
    public static string AWS_ACCESS_KEY_ID => Environment.GetEnvironmentVariable("AWS_ACCESS_KEY_ID");
    public static string AWS_SECRET_ACCESS_KEY => Environment.GetEnvironmentVariable("AWS_SECRET_ACCESS_KEY");
    public static string? AWS_SESSION_TOKEN => Environment.GetEnvironmentVariable("AWS_SESSION_TOKEN");
}

File: no-sdk/java/AwsCredentials.java
package sigv4.signing;

public class AwsCredentials {
    public static String getAwsAccessKeyId() {
        return System.getenv("AWS_ACCESS_KEY_ID");
    }

    public static String getAwsSecretAccessKey() {
        return System.getenv("AWS_SECRET_ACCESS_KEY");
    }

    public static String getAwsSessionToken() {
        return System.getenv("AWS_SESSION_TOKEN");
    }
}
